Core/clim-basic/setf-star.lisp

```diff
@@ -33,37 +33,51 @@
 			    (symbol-name '#:star))
 		    (symbol-package name-sym)))))
 
-(defun make-arglist (args)
-  "Convert the list ARGS to a list of arguments in the form (var
-init-form supplied-p-parameter)."
-  (mapcar (lambda (a) `(,a nil ,(gensym))) args))
-
 (defmacro defgeneric* (fun-name lambda-list &body options)
   "Defines a SETF* generic function.  FUN-NAME is a SETF function
 name.  The last required argument is the single argument to the function in a
 SETF place form; the other arguments are values collected from the
 SETF new value form."
   (unless (setf-name-p fun-name)
     (error "~S is not a valid name for a SETF* generic function." fun-name))
```

Comment:
Would be nice to use `with-current-source-form` for all syntax errors.

Reply:
for the sake of a kawaii world, I've wrapped both errors in the macro. fixed

Reply:
I had to withdraw this change -- utils are loaded /after/ setf-star file, so the macro is not recognized.

Reply:
We could at some point switch to `trivial-with-current-source-form` which is now in quicklisp. But that can also be done after this pull request (if at all).